docs/building.dox

```diff
@@ -124,6 +124,38 @@ cmake --build build --target docs
 
 The output website is located at `build/docs/html/`
 
+@section python-binding-builds Building Python Bindings
+
+This section contains instructions for building and installing the Python bindings, which are created with [pybind11](https://github.com/pybind/pybind11).
+
+## Steps
+
+1. Clone the `pyslang` repository (https://github.com/MikePopoloski/slang)
+2. Clone the `slang` submodule:
+
+@code{.ansi}
+git submodule update --init --recursive
+@endcode
+
+3. Create a virtual environment, activate it, and install `pyslang` as a Python package (including building the C++ `slang` library, with bindings):
```

Comment:
The venv isn't necessary, but I suppose it's fine to include it.

Reply:
Will change to indicate it's optional